integreat_cms/cms/templates/poicategories/poicategory_list_row.html

```diff
@@ -25,6 +25,16 @@
             </div>
         </div>
     </td>
+    <td class="pr-2">
+        {# djlint:off H006 #}
+        <img src="{{ poi_category.icon.url }}"
+             alt="{{ poi_category.icon.url }}"
+             class="max-h-12"/>
+        {# djlint:on #}
+    </td>
+    <td class="pr-2 text-right">{{ poi_category.color }}</td>
+    <td class="pr-2 text-right"
+        {% if poi_category.color %}bgcolor="{{ poi_category.color }}"{% endif %}></td>
```

Comment:
```suggestion
    <td class="py-2 pr-2 text-left flex items-center gap-2">
        {% if poi_category.color %}
            <div class="w-8 h-8 inline-block rounded-lg"
                 style="background-color: {{ poi_category.color }}"></div>
            <div>{{ poi_category.get_color_display }}</div>
        {% endif %}
    </td>
```
1. If you use `{{ poi_category.color }}` without any condition, it will print "None" when no color is set, which looks a bit weird
2. The attribute [`bgcolor`](https://www.geeksforgeeks.org/html-bgcolor-attribute/) isn't part of the HTML5 spec, so I would replace it by the style attribute.